.gitignore

```diff
@@ -178,3 +178,4 @@ unit_tests/base/test_base
 *.img
 *.vhd
 *.E01
+*.vmdk
```

Comment:
Does this work?  Looks like we downloading files to the tests/data folder, not unit_tests/base/